sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/models/LogsQueryException.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.query.models;
+
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.http.HttpResponse;
+
+/**
+ * Exception thrown when a query to retrieve logs fails.
+ */
+public final class LogsQueryException extends HttpResponseException {
```

Comment:
Curious - when do we try to create our own exception? Guidelines here said we should avoid creating new exceptions so in ACR I moved the generic exception to an inner exception.

Reply:
We create new exception types when `getValue()` needs to return a strongly-typed model type instead of just `Object` which `HttpResponseException` returns.